diff -urN FTGL/include/FTTextureGlyph.h FTGL.new/include/FTTextureGlyph.h
--- FTGL/include/FTTextureGlyph.h	2004-10-17 02:40:13.000000000 +0200
+++ FTGL.new/include/FTTextureGlyph.h	2006-04-12 23:51:45.000000000 +0200
@@ -52,7 +52,7 @@
          * Reset the currently active texture to zero to get into a known state before
          * drawing a string. This is to get round possible threading issues.
          */
-        static void FTTextureGlyph::ResetActiveTexture(){ activeTextureID = 0;}
+        static void ResetActiveTexture(){ activeTextureID = 0;}
         
     private:
         /**
